webhooks-extension/base/300-extension-deployment.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: webhooks-extension
+  namespace: tekton-pipelines
+  labels:
+    app: webhooks-extension
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: webhooks-extension
+  template:
+    metadata:
+      labels:
+        app: webhooks-extension
+    spec:
+      serviceAccountName: tekton-webhooks-extension
+      containers:
+        - name: webhooks-extension
+          image: webhooksExtensionImage
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+          livenessProbe:
+            httpGet:
+              path: /liveness
+              port: 8080
+          readinessProbe:
+            httpGet:
+              path: /readiness
+              port: 8080
+          env:
+          - name: PORT
+            value: "8080"
+          - name: INSTALLED_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: DOCKER_REGISTRY_LOCATION
+            value: DOCKER_REPO
+          - name: WEB_RESOURCES_DIR
+            value: web
+          - name: WEBHOOK_CALLBACK_URL
+            value: "http://listener.IPADDRESS.nip.io"
```

Comment:
Do you think we could eventually template this and change our recommended install instruction?